[2024-11-24T19:28:40.570+0000] {processor.py:161} INFO - Started process (PID=206) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:28:40.571+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:28:40.573+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:28:40.573+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:28:40.621+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:28:40.618+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/historical_data_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/historical_data_etl.py", line 89, in <module>
    cursor = return_snowflake_conn('snowflake_conn_nba')
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/historical_data_etl.py", line 14, in return_snowflake_conn
    conn = hook.get_conn()
           ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 286, in get_conn
    conn_config = self._get_conn_params()
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 185, in _get_conn_params
    conn = self.get_connection(self.snowflake_conn_id)  # type: ignore[attr-defined]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/hooks/base.py", line 83, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py", line 519, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `snowflake_conn_nba` isn't defined
[2024-11-24T19:28:40.635+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:28:40.697+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.131 seconds
[2024-11-24T19:29:13.475+0000] {processor.py:161} INFO - Started process (PID=263) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:29:13.476+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:29:13.478+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:29:13.477+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:29:13.500+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:29:13.497+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/historical_data_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/historical_data_etl.py", line 89, in <module>
    cursor = return_snowflake_conn('snowflake_conn_nba')
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/historical_data_etl.py", line 14, in return_snowflake_conn
    conn = hook.get_conn()
           ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 286, in get_conn
    conn_config = self._get_conn_params()
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 185, in _get_conn_params
    conn = self.get_connection(self.snowflake_conn_id)  # type: ignore[attr-defined]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/hooks/base.py", line 83, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py", line 519, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `snowflake_conn_nba` isn't defined
[2024-11-24T19:29:13.501+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:29:13.512+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.041 seconds
[2024-11-24T19:29:46.336+0000] {processor.py:161} INFO - Started process (PID=274) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:29:46.336+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:29:46.338+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:29:46.338+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:29:46.360+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:29:46.357+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/historical_data_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/historical_data_etl.py", line 89, in <module>
    cursor = return_snowflake_conn('snowflake_conn_nba')
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/historical_data_etl.py", line 14, in return_snowflake_conn
    conn = hook.get_conn()
           ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 286, in get_conn
    conn_config = self._get_conn_params()
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 185, in _get_conn_params
    conn = self.get_connection(self.snowflake_conn_id)  # type: ignore[attr-defined]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/hooks/base.py", line 83, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py", line 519, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `snowflake_conn_nba` isn't defined
[2024-11-24T19:29:46.361+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:29:46.368+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.037 seconds
[2024-11-24T19:30:18.200+0000] {processor.py:161} INFO - Started process (PID=285) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:30:18.201+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:30:18.203+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:30:18.202+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:30:18.224+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:30:18.221+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/historical_data_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/historical_data_etl.py", line 89, in <module>
    cursor = return_snowflake_conn('snowflake_conn_nba')
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/historical_data_etl.py", line 14, in return_snowflake_conn
    conn = hook.get_conn()
           ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 286, in get_conn
    conn_config = self._get_conn_params()
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 185, in _get_conn_params
    conn = self.get_connection(self.snowflake_conn_id)  # type: ignore[attr-defined]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/hooks/base.py", line 83, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py", line 519, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `snowflake_conn_nba` isn't defined
[2024-11-24T19:30:18.225+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:30:18.232+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.036 seconds
[2024-11-24T19:30:51.050+0000] {processor.py:161} INFO - Started process (PID=296) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:30:51.051+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:30:51.052+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:30:51.052+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:30:51.072+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:30:51.070+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/historical_data_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/historical_data_etl.py", line 89, in <module>
    cursor = return_snowflake_conn('snowflake_conn_nba')
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/historical_data_etl.py", line 14, in return_snowflake_conn
    conn = hook.get_conn()
           ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 286, in get_conn
    conn_config = self._get_conn_params()
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 185, in _get_conn_params
    conn = self.get_connection(self.snowflake_conn_id)  # type: ignore[attr-defined]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/hooks/base.py", line 83, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py", line 519, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `snowflake_conn_nba` isn't defined
[2024-11-24T19:30:51.073+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:30:51.081+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.035 seconds
[2024-11-24T19:31:23.917+0000] {processor.py:161} INFO - Started process (PID=307) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:31:23.918+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:31:23.920+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:31:23.920+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:31:23.943+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:31:23.940+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/historical_data_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/historical_data_etl.py", line 89, in <module>
    cursor = return_snowflake_conn('snowflake_conn_nba')
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/historical_data_etl.py", line 14, in return_snowflake_conn
    conn = hook.get_conn()
           ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 286, in get_conn
    conn_config = self._get_conn_params()
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 185, in _get_conn_params
    conn = self.get_connection(self.snowflake_conn_id)  # type: ignore[attr-defined]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/hooks/base.py", line 83, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py", line 519, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `snowflake_conn_nba` isn't defined
[2024-11-24T19:31:23.943+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:31:23.951+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.038 seconds
[2024-11-24T19:31:56.775+0000] {processor.py:161} INFO - Started process (PID=318) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:31:56.776+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:31:56.777+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:31:56.777+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:31:56.797+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:31:56.794+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/historical_data_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/historical_data_etl.py", line 89, in <module>
    cursor = return_snowflake_conn('snowflake_conn_nba')
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/historical_data_etl.py", line 14, in return_snowflake_conn
    conn = hook.get_conn()
           ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 286, in get_conn
    conn_config = self._get_conn_params()
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 185, in _get_conn_params
    conn = self.get_connection(self.snowflake_conn_id)  # type: ignore[attr-defined]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/hooks/base.py", line 83, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py", line 519, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `snowflake_conn_nba` isn't defined
[2024-11-24T19:31:56.798+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:31:56.806+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.034 seconds
[2024-11-24T19:32:29.655+0000] {processor.py:161} INFO - Started process (PID=329) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:32:29.656+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:32:29.658+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:32:29.658+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:32:29.695+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:32:29.690+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/historical_data_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/historical_data_etl.py", line 89, in <module>
    cursor = return_snowflake_conn('snowflake_conn_nba')
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/historical_data_etl.py", line 14, in return_snowflake_conn
    conn = hook.get_conn()
           ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 286, in get_conn
    conn_config = self._get_conn_params()
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 185, in _get_conn_params
    conn = self.get_connection(self.snowflake_conn_id)  # type: ignore[attr-defined]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/hooks/base.py", line 83, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py", line 519, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `snowflake_conn_nba` isn't defined
[2024-11-24T19:32:29.696+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:32:29.706+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.056 seconds
[2024-11-24T19:33:02.518+0000] {processor.py:161} INFO - Started process (PID=340) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:33:02.518+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:33:02.520+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:33:02.520+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:33:02.542+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:33:02.539+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/historical_data_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/historical_data_etl.py", line 89, in <module>
    cursor = return_snowflake_conn('snowflake_conn_nba')
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/historical_data_etl.py", line 14, in return_snowflake_conn
    conn = hook.get_conn()
           ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 286, in get_conn
    conn_config = self._get_conn_params()
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 185, in _get_conn_params
    conn = self.get_connection(self.snowflake_conn_id)  # type: ignore[attr-defined]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/hooks/base.py", line 83, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py", line 519, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `snowflake_conn_nba` isn't defined
[2024-11-24T19:33:02.542+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:33:02.550+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.037 seconds
[2024-11-24T19:33:35.412+0000] {processor.py:161} INFO - Started process (PID=351) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:33:35.413+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:33:35.415+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:33:35.414+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:33:35.437+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:33:35.434+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/historical_data_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/historical_data_etl.py", line 89, in <module>
    cursor = return_snowflake_conn('snowflake_conn_nba')
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/historical_data_etl.py", line 14, in return_snowflake_conn
    conn = hook.get_conn()
           ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 286, in get_conn
    conn_config = self._get_conn_params()
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 185, in _get_conn_params
    conn = self.get_connection(self.snowflake_conn_id)  # type: ignore[attr-defined]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/hooks/base.py", line 83, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py", line 519, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `snowflake_conn_nba` isn't defined
[2024-11-24T19:33:35.437+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:33:35.446+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.039 seconds
[2024-11-24T19:34:07.285+0000] {processor.py:161} INFO - Started process (PID=362) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:34:07.285+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:34:07.287+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:34:07.286+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:34:07.307+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:34:07.304+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/historical_data_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/historical_data_etl.py", line 89, in <module>
    cursor = return_snowflake_conn('snowflake_conn_nba')
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/historical_data_etl.py", line 14, in return_snowflake_conn
    conn = hook.get_conn()
           ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 286, in get_conn
    conn_config = self._get_conn_params()
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 185, in _get_conn_params
    conn = self.get_connection(self.snowflake_conn_id)  # type: ignore[attr-defined]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/hooks/base.py", line 83, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py", line 519, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `snowflake_conn_nba` isn't defined
[2024-11-24T19:34:07.307+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:34:07.315+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.034 seconds
[2024-11-24T19:34:40.158+0000] {processor.py:161} INFO - Started process (PID=373) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:34:40.158+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:34:40.160+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:34:40.160+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:34:40.180+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:34:40.177+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/historical_data_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/historical_data_etl.py", line 89, in <module>
    cursor = return_snowflake_conn('snowflake_conn_nba')
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/historical_data_etl.py", line 14, in return_snowflake_conn
    conn = hook.get_conn()
           ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 286, in get_conn
    conn_config = self._get_conn_params()
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 185, in _get_conn_params
    conn = self.get_connection(self.snowflake_conn_id)  # type: ignore[attr-defined]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/hooks/base.py", line 83, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py", line 519, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `snowflake_conn_nba` isn't defined
[2024-11-24T19:34:40.180+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:34:40.188+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.034 seconds
[2024-11-24T19:35:13.032+0000] {processor.py:161} INFO - Started process (PID=384) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:35:13.032+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:35:13.034+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:35:13.033+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:35:13.053+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:35:13.051+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/historical_data_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/historical_data_etl.py", line 89, in <module>
    cursor = return_snowflake_conn('snowflake_conn_nba')
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/historical_data_etl.py", line 14, in return_snowflake_conn
    conn = hook.get_conn()
           ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 286, in get_conn
    conn_config = self._get_conn_params()
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 185, in _get_conn_params
    conn = self.get_connection(self.snowflake_conn_id)  # type: ignore[attr-defined]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/hooks/base.py", line 83, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py", line 519, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `snowflake_conn_nba` isn't defined
[2024-11-24T19:35:13.054+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:35:13.061+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.034 seconds
[2024-11-24T19:35:45.898+0000] {processor.py:161} INFO - Started process (PID=395) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:35:45.898+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:35:45.900+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:35:45.900+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:35:45.925+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:35:45.922+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/historical_data_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/historical_data_etl.py", line 89, in <module>
    cursor = return_snowflake_conn('snowflake_conn_nba')
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/historical_data_etl.py", line 14, in return_snowflake_conn
    conn = hook.get_conn()
           ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 286, in get_conn
    conn_config = self._get_conn_params()
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 185, in _get_conn_params
    conn = self.get_connection(self.snowflake_conn_id)  # type: ignore[attr-defined]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/hooks/base.py", line 83, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py", line 519, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `snowflake_conn_nba` isn't defined
[2024-11-24T19:35:45.926+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:35:45.934+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.041 seconds
[2024-11-24T19:36:18.753+0000] {processor.py:161} INFO - Started process (PID=406) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:36:18.754+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:36:18.755+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:36:18.755+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:36:18.776+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:36:18.773+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/historical_data_etl.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/historical_data_etl.py", line 89, in <module>
    cursor = return_snowflake_conn('snowflake_conn_nba')
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/historical_data_etl.py", line 14, in return_snowflake_conn
    conn = hook.get_conn()
           ^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 286, in get_conn
    conn_config = self._get_conn_params()
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py", line 185, in _get_conn_params
    conn = self.get_connection(self.snowflake_conn_id)  # type: ignore[attr-defined]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/hooks/base.py", line 83, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py", line 519, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `snowflake_conn_nba` isn't defined
[2024-11-24T19:36:18.776+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:36:18.784+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.035 seconds
[2024-11-24T19:36:51.636+0000] {processor.py:161} INFO - Started process (PID=417) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:36:51.636+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:36:51.638+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:36:51.638+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:36:51.657+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:36:51.657+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:36:51.658+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:36:51.658+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:36:51.659+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:36:51.659+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:36:52.560+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:36:52.634+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:36:52.634+0000] {override.py:1829} INFO - Created Permission View: can read on DAG:nba_games_etl
[2024-11-24T19:36:52.641+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:36:52.641+0000] {override.py:1829} INFO - Created Permission View: can edit on DAG:nba_games_etl
[2024-11-24T19:36:52.646+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:36:52.646+0000] {override.py:1829} INFO - Created Permission View: can delete on DAG:nba_games_etl
[2024-11-24T19:36:52.647+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:36:52.646+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:36:52.654+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:36:52.654+0000] {dag.py:3118} INFO - Creating ORM DAG for nba_games_etl
[2024-11-24T19:36:52.655+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:36:52.655+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:36:52.669+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 1.038 seconds
[2024-11-24T19:37:24.492+0000] {processor.py:161} INFO - Started process (PID=428) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:37:24.492+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:37:24.494+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:37:24.494+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:37:24.511+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:37:24.511+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:37:24.512+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:37:24.512+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:37:24.513+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:37:24.513+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:37:25.013+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:37:25.074+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:37:25.074+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:37:25.081+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:37:25.081+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:37:25.096+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.608 seconds
[2024-11-24T19:37:56.353+0000] {processor.py:161} INFO - Started process (PID=439) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:37:56.354+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:37:56.356+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:37:56.355+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:37:56.376+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:37:56.376+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:37:56.377+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:37:56.377+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:37:56.378+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:37:56.378+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:37:56.820+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:37:56.885+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:37:56.884+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:37:56.892+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:37:56.892+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:37:56.908+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.559 seconds
[2024-11-24T19:38:29.204+0000] {processor.py:161} INFO - Started process (PID=450) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:38:29.205+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:38:29.207+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:38:29.206+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:38:29.228+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:38:29.228+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:38:29.229+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:38:29.229+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:38:29.230+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:38:29.230+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:38:29.732+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:38:29.800+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:38:29.800+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:38:29.808+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:38:29.808+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:38:29.822+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.623 seconds
[2024-11-24T19:39:02.642+0000] {processor.py:161} INFO - Started process (PID=461) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:39:02.643+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:39:02.644+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:39:02.644+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:39:02.664+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:39:02.663+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:39:02.664+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:39:02.664+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:39:02.665+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:39:02.665+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:39:03.118+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:39:03.183+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:39:03.182+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:39:03.190+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:39:03.190+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:39:03.205+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.567 seconds
[2024-11-24T19:39:35.090+0000] {processor.py:161} INFO - Started process (PID=472) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:39:35.091+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:39:35.094+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:39:35.092+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:39:35.186+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:39:35.186+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:39:35.187+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:39:35.187+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:39:35.188+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:39:35.188+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:39:35.986+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:39:36.128+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:39:36.127+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:39:36.173+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:39:36.173+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:39:36.189+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 1.108 seconds
[2024-11-24T19:40:18.455+0000] {processor.py:161} INFO - Started process (PID=207) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:40:18.456+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:40:18.458+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:40:18.458+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:40:18.483+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:40:18.482+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:40:18.484+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:40:18.484+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:40:18.485+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:40:18.484+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:40:19.033+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:40:19.123+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:40:19.122+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:40:19.136+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:40:19.136+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:40:19.153+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.704 seconds
[2024-11-24T19:40:49.314+0000] {processor.py:161} INFO - Started process (PID=251) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:40:49.315+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:40:49.317+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:40:49.316+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:40:49.340+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:40:49.339+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:40:49.340+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:40:49.340+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:40:49.341+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:40:49.341+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:40:49.888+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:40:49.955+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:40:49.955+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:40:49.962+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:40:49.962+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:40:49.978+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.670 seconds
[2024-11-24T19:41:20.145+0000] {processor.py:161} INFO - Started process (PID=262) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:41:20.146+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:41:20.148+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:41:20.148+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:41:20.170+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:41:20.170+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:41:20.172+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:41:20.171+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:41:20.172+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:41:20.172+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:41:20.678+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:41:20.783+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:41:20.782+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:41:20.790+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:41:20.789+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:41:20.805+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.668 seconds
[2024-11-24T19:41:50.974+0000] {processor.py:161} INFO - Started process (PID=273) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:41:50.974+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:41:50.976+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:41:50.976+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:41:50.995+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:41:50.995+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:41:50.996+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:41:50.996+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:41:50.996+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:41:50.996+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:41:51.449+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:41:51.510+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:41:51.510+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:41:51.518+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:41:51.518+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:41:51.533+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.563 seconds
[2024-11-24T19:42:21.806+0000] {processor.py:161} INFO - Started process (PID=284) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:42:21.807+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:42:21.808+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:42:21.808+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:42:21.829+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:42:21.829+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:42:21.830+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:42:21.830+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:42:21.831+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:42:21.831+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:42:22.277+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:42:22.338+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:42:22.338+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:42:22.345+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:42:22.345+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:42:22.359+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.558 seconds
[2024-11-24T19:42:52.614+0000] {processor.py:161} INFO - Started process (PID=295) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:42:52.615+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:42:52.616+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:42:52.616+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:42:52.636+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:42:52.636+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:42:52.637+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:42:52.637+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:42:52.638+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:42:52.638+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:42:53.216+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:42:53.278+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:42:53.277+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:42:53.285+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:42:53.284+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:42:53.300+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.691 seconds
[2024-11-24T19:43:24.049+0000] {processor.py:161} INFO - Started process (PID=314) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:43:24.050+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:43:24.052+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:43:24.052+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:43:24.071+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:43:24.071+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:43:24.072+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:43:24.072+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:43:24.073+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:43:24.073+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:43:24.504+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:43:24.519+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:43:24.519+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:43:24.529+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:43:24.528+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:43:24.542+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.497 seconds
[2024-11-24T19:43:56.403+0000] {processor.py:161} INFO - Started process (PID=325) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:43:56.404+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:43:56.405+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:43:56.405+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:43:56.425+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:43:56.425+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:43:56.426+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:43:56.426+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:43:56.427+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:43:56.427+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:43:56.875+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:43:56.977+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:43:56.976+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:43:56.984+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:43:56.984+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:43:57.000+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.601 seconds
[2024-11-24T19:45:41.689+0000] {processor.py:161} INFO - Started process (PID=342) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:45:41.690+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:45:41.692+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:45:41.691+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:45:41.710+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:45:41.710+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:45:41.711+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:45:41.711+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:45:41.712+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:45:41.712+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:45:42.455+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:45:42.538+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:45:42.538+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:45:42.547+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:45:42.547+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:45:42.561+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.875 seconds
[2024-11-24T19:46:13.427+0000] {processor.py:161} INFO - Started process (PID=379) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:46:13.428+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:46:13.429+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:46:13.429+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:46:13.449+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:46:13.449+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:46:13.450+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:46:13.450+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:46:13.451+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:46:13.451+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:46:13.972+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:46:14.035+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:46:14.034+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:46:14.042+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:46:14.041+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:46:14.055+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.633 seconds
[2024-11-24T19:46:44.399+0000] {processor.py:161} INFO - Started process (PID=390) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:46:44.400+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:46:44.401+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:46:44.401+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:46:44.420+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:46:44.420+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:46:44.421+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:46:44.421+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:46:44.422+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:46:44.422+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:46:44.995+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:46:45.059+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:46:45.058+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:46:45.065+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:46:45.065+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:46:45.080+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.685 seconds
[2024-11-24T19:47:15.778+0000] {processor.py:161} INFO - Started process (PID=402) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:47:15.779+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:47:15.781+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:47:15.781+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:47:15.804+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:47:15.804+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:47:15.806+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:47:15.805+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:47:15.807+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:47:15.807+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:47:16.494+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:47:16.559+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:47:16.559+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:47:16.566+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:47:16.566+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:47:16.583+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.809 seconds
[2024-11-24T19:47:46.750+0000] {processor.py:161} INFO - Started process (PID=419) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:47:46.750+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:47:46.752+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:47:46.752+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:47:46.776+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:47:46.776+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:47:46.777+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:47:46.777+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:47:46.778+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:47:46.778+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:47:47.943+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:47:48.012+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:47:48.011+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:47:48.019+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:47:48.019+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:47:48.034+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 1.289 seconds
[2024-11-24T19:48:18.602+0000] {processor.py:161} INFO - Started process (PID=430) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:48:18.603+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:48:18.605+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:48:18.605+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:48:18.626+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:48:18.626+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:48:18.627+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:48:18.627+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:48:18.628+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:48:18.628+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:48:19.169+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:48:19.238+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:48:19.238+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:48:19.246+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:48:19.246+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:48:19.261+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.663 seconds
[2024-11-24T19:48:51.138+0000] {processor.py:161} INFO - Started process (PID=441) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:48:51.139+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:48:51.140+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:48:51.140+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:48:51.160+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:48:51.160+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:48:51.161+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:48:51.161+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:48:51.162+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:48:51.162+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:48:51.692+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:48:51.757+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:48:51.756+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:48:51.766+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:48:51.765+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:48:51.781+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.648 seconds
[2024-11-24T19:49:11.839+0000] {processor.py:161} INFO - Started process (PID=452) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:49:11.840+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:49:11.842+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:49:11.841+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:49:11.862+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:49:11.862+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:49:11.863+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:49:11.863+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:49:11.864+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:49:11.864+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:49:12.430+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:49:12.434+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:49:12.433+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:49:12.442+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:49:12.442+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:49:12.460+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.625 seconds
[2024-11-24T19:49:42.704+0000] {processor.py:161} INFO - Started process (PID=463) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:49:42.705+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:49:42.706+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:49:42.706+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:49:42.727+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:49:42.726+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:49:42.727+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:49:42.727+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:49:42.728+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:49:42.728+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:49:43.340+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:49:43.403+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:49:43.403+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:49:43.410+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:49:43.410+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:49:43.427+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.726 seconds
[2024-11-24T19:50:14.170+0000] {processor.py:161} INFO - Started process (PID=478) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:50:14.170+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:50:14.172+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:50:14.172+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:50:14.193+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:50:14.193+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:50:14.194+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:50:14.194+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:50:14.195+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:50:14.195+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:50:14.509+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:50:14.575+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:50:14.574+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:50:14.582+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:50:14.582+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:50:14.597+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.432 seconds
[2024-11-24T19:50:45.132+0000] {processor.py:161} INFO - Started process (PID=489) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:50:45.133+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:50:45.135+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:50:45.135+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:50:45.158+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:50:45.158+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:50:45.159+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:50:45.159+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:50:45.161+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:50:45.160+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:50:45.760+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:50:45.825+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:50:45.825+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:50:45.832+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:50:45.832+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:50:45.848+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.721 seconds
[2024-11-24T19:51:16.288+0000] {processor.py:161} INFO - Started process (PID=500) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:51:16.289+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:51:16.291+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:51:16.291+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:51:16.312+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:51:16.311+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:51:16.313+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:51:16.312+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:51:16.313+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:51:16.313+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:51:16.842+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:51:16.904+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:51:16.904+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:51:16.912+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:51:16.911+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:51:16.928+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.644 seconds
[2024-11-24T19:51:47.557+0000] {processor.py:161} INFO - Started process (PID=511) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:51:47.558+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:51:47.560+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:51:47.559+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:51:47.580+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:51:47.579+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:51:47.581+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:51:47.580+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:51:47.581+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:51:47.581+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:51:48.166+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:51:48.229+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:51:48.229+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:51:48.237+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:51:48.237+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:51:48.252+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.699 seconds
[2024-11-24T19:52:18.629+0000] {processor.py:161} INFO - Started process (PID=522) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:52:18.630+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:52:18.632+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:52:18.631+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:52:18.651+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:52:18.651+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:52:18.652+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:52:18.652+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:52:18.653+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:52:18.653+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:52:19.214+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:52:19.275+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:52:19.275+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:52:19.283+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:52:19.283+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:52:19.298+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.673 seconds
[2024-11-24T19:52:49.901+0000] {processor.py:161} INFO - Started process (PID=533) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:52:49.902+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:52:49.903+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:52:49.903+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:52:49.924+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:52:49.924+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:52:49.925+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:52:49.925+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:52:49.926+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:52:49.926+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:52:50.512+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:52:50.574+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:52:50.574+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:52:50.581+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:52:50.581+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:52:50.594+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.697 seconds
[2024-11-24T19:53:20.989+0000] {processor.py:161} INFO - Started process (PID=554) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:53:20.989+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:53:20.991+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:53:20.991+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:53:21.012+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:53:21.011+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:53:21.013+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:53:21.012+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:53:21.014+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:53:21.014+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:53:21.802+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:53:21.866+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:53:21.865+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:53:21.873+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:53:21.873+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:53:21.889+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.905 seconds
[2024-11-24T19:53:52.341+0000] {processor.py:161} INFO - Started process (PID=565) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:53:52.342+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:53:52.343+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:53:52.343+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:53:52.364+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:53:52.364+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:53:52.365+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:53:52.365+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:53:52.366+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:53:52.366+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:53:52.911+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:53:52.982+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:53:52.982+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:53:52.989+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:53:52.989+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:53:53.003+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.666 seconds
[2024-11-24T19:54:23.582+0000] {processor.py:161} INFO - Started process (PID=576) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:54:23.583+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:54:23.584+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:54:23.584+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:54:23.606+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:54:23.606+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:54:23.607+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:54:23.607+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:54:23.608+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:54:23.608+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:54:24.267+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:54:24.351+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:54:24.350+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:54:24.360+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:54:24.360+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:54:24.378+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.801 seconds
[2024-11-24T19:54:54.693+0000] {processor.py:161} INFO - Started process (PID=587) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:54:54.693+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:54:54.695+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:54:54.695+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:54:54.714+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:54:54.714+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:54:54.715+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:54:54.715+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:54:54.716+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:54:54.716+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:54:55.214+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:54:55.275+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:54:55.275+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:54:55.283+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:54:55.283+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:54:55.298+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.609 seconds
[2024-11-24T19:55:26.099+0000] {processor.py:161} INFO - Started process (PID=592) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:55:26.100+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:55:26.101+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:55:26.101+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:55:26.121+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:55:26.121+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:55:26.122+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:55:26.122+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:55:26.123+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:55:26.123+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:55:26.616+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:55:26.677+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:55:26.677+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:55:26.684+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:55:26.684+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:55:26.697+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.602 seconds
[2024-11-24T19:55:56.993+0000] {processor.py:161} INFO - Started process (PID=603) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:55:56.994+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:55:56.996+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:55:56.995+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:55:57.016+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:55:57.016+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:55:57.017+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:55:57.017+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:55:57.018+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:55:57.018+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:55:57.573+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:55:57.637+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:55:57.636+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:55:57.644+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:55:57.644+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:55:57.659+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.671 seconds
[2024-11-24T19:56:29.387+0000] {processor.py:161} INFO - Started process (PID=614) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:56:29.388+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:56:29.390+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:56:29.390+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:56:29.409+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:56:29.409+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:56:29.410+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:56:29.410+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:56:29.411+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:56:29.411+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:56:29.930+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:56:29.996+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:56:29.995+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:56:30.003+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:56:30.003+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:56:30.019+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.636 seconds
[2024-11-24T19:57:01.876+0000] {processor.py:161} INFO - Started process (PID=625) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:57:01.877+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:57:01.878+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:57:01.878+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:57:01.900+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:57:01.900+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:57:01.901+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:57:01.901+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:57:01.902+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:57:01.902+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:57:02.684+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:57:02.787+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:57:02.786+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:57:02.794+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:57:02.794+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:57:02.810+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.940 seconds
[2024-11-24T19:57:34.685+0000] {processor.py:161} INFO - Started process (PID=636) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:57:34.686+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:57:34.687+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:57:34.687+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:57:34.708+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:57:34.708+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:57:34.710+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:57:34.709+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:57:34.710+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:57:34.710+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:57:35.233+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:57:35.311+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:57:35.311+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:57:35.320+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:57:35.319+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:57:35.339+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.659 seconds
[2024-11-24T19:58:08.020+0000] {processor.py:161} INFO - Started process (PID=647) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:58:08.021+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:58:08.023+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:58:08.022+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:58:08.048+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:58:08.048+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:58:08.049+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:58:08.049+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:58:08.051+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:58:08.051+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:58:08.588+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:58:08.656+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:58:08.656+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:58:08.664+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:58:08.664+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:58:08.678+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.662 seconds
[2024-11-24T19:58:40.546+0000] {processor.py:161} INFO - Started process (PID=659) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:58:40.546+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:58:40.548+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:58:40.548+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:58:40.566+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:58:40.566+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:58:40.567+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:58:40.567+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:58:40.568+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:58:40.568+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:58:41.193+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:58:41.257+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:58:41.257+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:58:41.264+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:58:41.264+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:58:41.280+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.738 seconds
[2024-11-24T19:59:12.863+0000] {processor.py:161} INFO - Started process (PID=670) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:59:12.864+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:59:12.866+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:59:12.865+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:59:12.886+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:59:12.885+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:59:12.887+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:59:12.886+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:59:12.888+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:59:12.887+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:59:13.574+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:59:13.636+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:59:13.636+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:59:13.643+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:59:13.643+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:59:13.659+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.800 seconds
[2024-11-24T19:59:45.537+0000] {processor.py:161} INFO - Started process (PID=681) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:59:45.539+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T19:59:45.542+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:59:45.542+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:59:45.569+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:59:45.568+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T19:59:45.570+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:59:45.570+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T19:59:45.571+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:59:45.571+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T19:59:46.148+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T19:59:46.214+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:59:46.213+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T19:59:46.222+0000] {logging_mixin.py:188} INFO - [2024-11-24T19:59:46.221+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T19:59:46.237+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.706 seconds
[2024-11-24T20:00:18.807+0000] {processor.py:161} INFO - Started process (PID=692) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:00:18.807+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:00:18.809+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:00:18.809+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:00:18.829+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:00:18.829+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:00:18.830+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:00:18.830+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:00:18.831+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:00:18.831+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:00:19.349+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:00:19.414+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:00:19.413+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:00:19.421+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:00:19.420+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:00:19.435+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.633 seconds
[2024-11-24T20:00:51.328+0000] {processor.py:161} INFO - Started process (PID=703) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:00:51.328+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:00:51.330+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:00:51.329+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:00:51.350+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:00:51.350+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:00:51.351+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:00:51.351+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:00:51.352+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:00:51.352+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:00:51.789+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:00:51.853+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:00:51.852+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:00:51.861+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:00:51.860+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:00:51.877+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.554 seconds
[2024-11-24T20:01:24.062+0000] {processor.py:161} INFO - Started process (PID=714) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:01:24.063+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:01:24.065+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:01:24.064+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:01:24.084+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:01:24.084+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:01:24.085+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:01:24.085+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:01:24.086+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:01:24.086+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:01:24.677+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:01:24.740+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:01:24.740+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:01:24.748+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:01:24.748+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:01:24.764+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.706 seconds
[2024-11-24T20:01:56.589+0000] {processor.py:161} INFO - Started process (PID=725) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:01:56.590+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:01:56.591+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:01:56.591+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:01:56.611+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:01:56.611+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:01:56.612+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:01:56.612+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:01:56.613+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:01:56.613+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:01:57.190+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:01:57.260+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:01:57.260+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:01:57.270+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:01:57.269+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:01:57.289+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.705 seconds
[2024-11-24T20:02:29.159+0000] {processor.py:161} INFO - Started process (PID=736) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:02:29.160+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:02:29.161+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:02:29.161+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:02:29.183+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:02:29.183+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:02:29.184+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:02:29.184+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:02:29.185+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:02:29.184+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:02:29.718+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:02:29.779+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:02:29.779+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:02:29.786+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:02:29.786+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:02:29.800+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.647 seconds
[2024-11-24T20:03:02.423+0000] {processor.py:161} INFO - Started process (PID=747) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:03:02.424+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:03:02.425+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:03:02.425+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:03:02.445+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:03:02.445+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:03:02.446+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:03:02.446+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:03:02.447+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:03:02.447+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:03:02.947+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:03:03.017+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:03:03.016+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:03:03.024+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:03:03.024+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:03:03.040+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.621 seconds
[2024-11-24T20:03:34.894+0000] {processor.py:161} INFO - Started process (PID=758) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:03:34.895+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:03:34.897+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:03:34.896+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:03:34.916+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:03:34.916+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:03:34.917+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:03:34.917+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:03:34.918+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:03:34.918+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:03:35.382+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:03:35.445+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:03:35.445+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:03:35.452+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:03:35.452+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:03:35.468+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.578 seconds
[2024-11-24T20:04:07.343+0000] {processor.py:161} INFO - Started process (PID=769) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:04:07.344+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:04:07.346+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:04:07.345+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:04:07.365+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:04:07.365+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:04:07.366+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:04:07.366+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:04:07.367+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:04:07.367+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:04:07.961+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:04:08.027+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:04:08.027+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:04:08.034+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:04:08.034+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:04:08.051+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.712 seconds
[2024-11-24T20:04:40.579+0000] {processor.py:161} INFO - Started process (PID=780) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:04:40.580+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:04:40.582+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:04:40.581+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:04:40.603+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:04:40.602+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:04:40.604+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:04:40.603+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:04:40.605+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:04:40.604+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:04:41.078+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:04:41.146+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:04:41.145+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:04:41.155+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:04:41.154+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:04:41.170+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.596 seconds
[2024-11-24T20:05:13.031+0000] {processor.py:161} INFO - Started process (PID=791) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:05:13.031+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:05:13.033+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:05:13.033+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:05:13.052+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:05:13.052+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:05:13.053+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:05:13.053+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:05:13.054+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:05:13.054+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:05:13.458+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:05:13.533+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:05:13.533+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:05:13.543+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:05:13.542+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:05:13.569+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.543 seconds
[2024-11-24T20:05:45.875+0000] {processor.py:161} INFO - Started process (PID=802) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:05:45.875+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:05:45.877+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:05:45.877+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:05:45.897+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:05:45.897+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:05:45.898+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:05:45.898+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:05:45.899+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:05:45.899+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:05:46.369+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:05:46.431+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:05:46.431+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:05:46.438+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:05:46.438+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:05:46.452+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.582 seconds
[2024-11-24T20:06:18.293+0000] {processor.py:161} INFO - Started process (PID=813) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:06:18.294+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:06:18.295+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:06:18.295+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:06:18.313+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:06:18.313+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:06:18.314+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:06:18.314+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:06:18.315+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:06:18.315+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:06:18.966+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:06:19.029+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:06:19.029+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:06:19.037+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:06:19.036+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:06:19.050+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.761 seconds
[2024-11-24T20:06:51.123+0000] {processor.py:161} INFO - Started process (PID=824) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:06:51.124+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:06:51.125+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:06:51.125+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:06:51.144+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:06:51.144+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:06:51.145+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:06:51.145+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:06:51.146+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:06:51.146+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:06:51.785+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:06:51.852+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:06:51.852+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:06:51.859+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:06:51.859+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:06:51.876+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.757 seconds
[2024-11-24T20:07:23.753+0000] {processor.py:161} INFO - Started process (PID=835) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:07:23.754+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:07:23.756+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:07:23.756+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:07:23.775+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:07:23.774+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:07:23.776+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:07:23.776+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:07:23.777+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:07:23.777+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:07:24.292+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:07:24.353+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:07:24.353+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:07:24.360+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:07:24.360+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:07:24.376+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.627 seconds
[2024-11-24T20:07:56.822+0000] {processor.py:161} INFO - Started process (PID=846) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:07:56.823+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:07:56.824+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:07:56.824+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:07:56.843+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:07:56.843+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:07:56.844+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:07:56.844+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:07:56.845+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:07:56.845+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:07:57.360+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:07:57.420+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:07:57.419+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:07:57.427+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:07:57.427+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:07:57.443+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.625 seconds
[2024-11-24T20:08:29.303+0000] {processor.py:161} INFO - Started process (PID=857) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:08:29.304+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:08:29.305+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:08:29.305+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:08:29.326+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:08:29.326+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:08:29.327+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:08:29.327+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:08:29.328+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:08:29.328+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:08:29.899+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:08:29.967+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:08:29.967+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:08:29.976+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:08:29.976+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:08:29.992+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.693 seconds
[2024-11-24T20:09:02.139+0000] {processor.py:161} INFO - Started process (PID=868) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:09:02.140+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:09:02.142+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:09:02.142+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:09:02.161+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:09:02.161+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:09:02.162+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:09:02.162+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:09:02.163+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:09:02.163+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:09:02.758+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:09:02.824+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:09:02.823+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:09:02.831+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:09:02.830+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:09:02.845+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.710 seconds
[2024-11-24T20:09:34.729+0000] {processor.py:161} INFO - Started process (PID=879) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:09:34.730+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:09:34.732+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:09:34.731+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:09:34.750+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:09:34.749+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:09:34.750+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:09:34.750+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:09:34.751+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:09:34.751+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:09:35.372+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:09:35.432+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:09:35.432+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:09:35.439+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:09:35.439+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:09:35.456+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.730 seconds
[2024-11-24T20:10:07.557+0000] {processor.py:161} INFO - Started process (PID=890) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:10:07.558+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:10:07.559+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:10:07.559+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:10:07.579+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:10:07.579+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:10:07.580+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:10:07.580+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:10:07.581+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:10:07.581+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:10:08.072+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:10:08.145+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:10:08.144+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:10:08.153+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:10:08.152+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:10:08.169+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.616 seconds
[2024-11-24T20:10:40.397+0000] {processor.py:161} INFO - Started process (PID=901) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:10:40.398+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:10:40.400+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:10:40.399+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:10:40.419+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:10:40.418+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:10:40.419+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:10:40.419+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:10:40.420+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:10:40.420+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:10:40.993+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:10:41.057+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:10:41.056+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:10:41.064+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:10:41.064+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:10:41.083+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.690 seconds
[2024-11-24T20:11:12.900+0000] {processor.py:161} INFO - Started process (PID=912) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:11:12.901+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:11:12.903+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:11:12.902+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:11:12.921+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:11:12.921+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:11:12.922+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:11:12.922+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:11:12.923+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:11:12.923+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:11:13.472+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:11:13.534+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:11:13.534+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:11:13.543+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:11:13.542+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:11:13.556+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.660 seconds
[2024-11-24T20:11:45.769+0000] {processor.py:161} INFO - Started process (PID=923) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:11:45.770+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:11:45.771+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:11:45.771+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:11:45.791+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:11:45.791+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:11:45.792+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:11:45.792+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:11:45.793+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:11:45.793+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:11:46.423+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:11:46.485+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:11:46.484+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:11:46.492+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:11:46.492+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:11:46.506+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.741 seconds
[2024-11-24T20:12:18.281+0000] {processor.py:161} INFO - Started process (PID=935) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:12:18.282+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:12:18.283+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:12:18.283+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:12:18.303+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:12:18.303+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:12:18.304+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:12:18.304+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:12:18.305+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:12:18.305+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:12:18.793+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:12:18.860+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:12:18.860+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:12:18.868+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:12:18.868+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:12:18.883+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.607 seconds
[2024-11-24T20:12:51.204+0000] {processor.py:161} INFO - Started process (PID=946) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:12:51.205+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:12:51.206+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:12:51.206+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:12:51.226+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:12:51.226+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:12:51.227+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:12:51.227+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:12:51.228+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:12:51.228+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:12:51.802+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:12:51.867+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:12:51.867+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:12:51.875+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:12:51.875+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:12:51.893+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.693 seconds
[2024-11-24T20:13:24.476+0000] {processor.py:161} INFO - Started process (PID=957) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:13:24.477+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:13:24.478+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:13:24.478+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:13:24.499+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:13:24.499+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:13:24.500+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:13:24.500+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:13:24.501+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:13:24.501+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:13:24.931+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:13:24.995+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:13:24.995+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:13:25.002+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:13:25.002+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:13:25.017+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.546 seconds
[2024-11-24T20:13:56.604+0000] {processor.py:161} INFO - Started process (PID=968) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:13:56.606+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:13:56.607+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:13:56.607+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:13:56.633+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:13:56.633+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:13:56.634+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:13:56.634+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:13:56.635+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:13:56.635+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:13:57.095+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:13:57.160+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:13:57.159+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:13:57.167+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:13:57.167+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:13:57.183+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.584 seconds
[2024-11-24T20:14:29.868+0000] {processor.py:161} INFO - Started process (PID=979) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:14:29.868+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:14:29.870+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:14:29.870+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:14:29.890+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:14:29.890+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:14:29.891+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:14:29.891+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:14:29.893+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:14:29.892+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:14:30.370+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:14:30.430+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:14:30.429+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:14:30.437+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:14:30.437+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:14:30.453+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.589 seconds
[2024-11-24T20:15:02.316+0000] {processor.py:161} INFO - Started process (PID=990) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:15:02.317+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:15:02.319+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:15:02.318+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:15:02.339+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:15:02.339+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:15:02.340+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:15:02.340+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:15:02.341+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:15:02.341+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:15:02.876+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:15:02.939+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:15:02.938+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:15:02.946+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:15:02.945+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:15:02.961+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.650 seconds
[2024-11-24T20:15:34.732+0000] {processor.py:161} INFO - Started process (PID=1001) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:15:34.733+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:15:34.735+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:15:34.734+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:15:34.753+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:15:34.753+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:15:34.754+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:15:34.754+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:15:34.755+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:15:34.755+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:15:35.332+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:15:35.394+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:15:35.393+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:15:35.401+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:15:35.401+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:15:35.415+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.687 seconds
[2024-11-24T20:16:07.665+0000] {processor.py:161} INFO - Started process (PID=1012) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:16:07.666+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:16:07.668+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:16:07.667+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:16:07.687+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:16:07.687+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:16:07.688+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:16:07.688+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:16:07.689+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:16:07.689+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:16:08.176+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:16:08.236+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:16:08.235+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:16:08.243+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:16:08.243+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:16:08.258+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.597 seconds
[2024-11-24T20:16:40.120+0000] {processor.py:161} INFO - Started process (PID=1023) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:16:40.121+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:16:40.122+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:16:40.122+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:16:40.141+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:16:40.141+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:16:40.142+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:16:40.142+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:16:40.143+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:16:40.143+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:16:40.629+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:16:40.690+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:16:40.690+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:16:40.697+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:16:40.697+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:16:40.714+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.599 seconds
[2024-11-24T20:17:13.355+0000] {processor.py:161} INFO - Started process (PID=1034) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:17:13.355+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:17:13.357+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:17:13.357+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:17:13.382+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:17:13.382+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:17:13.383+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:17:13.383+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:17:13.384+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:17:13.384+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:17:13.835+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:17:13.898+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:17:13.898+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:17:13.906+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:17:13.906+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:17:13.925+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.576 seconds
[2024-11-24T20:17:45.790+0000] {processor.py:161} INFO - Started process (PID=1045) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:17:45.790+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:17:45.792+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:17:45.792+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:17:45.811+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:17:45.811+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:17:45.812+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:17:45.812+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:17:45.813+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:17:45.813+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:17:46.231+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:17:46.292+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:17:46.292+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:17:46.299+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:17:46.299+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:17:46.316+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.530 seconds
[2024-11-24T20:18:18.639+0000] {processor.py:161} INFO - Started process (PID=1056) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:18:18.640+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:18:18.643+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:18:18.642+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:18:18.667+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:18:18.667+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:18:18.668+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:18:18.668+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:18:18.669+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:18:18.669+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:18:19.113+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:18:19.178+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:18:19.178+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:18:19.186+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:18:19.186+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:18:19.204+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.570 seconds
[2024-11-24T20:18:51.032+0000] {processor.py:161} INFO - Started process (PID=1067) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:18:51.033+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:18:51.035+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:18:51.034+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:18:51.057+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:18:51.056+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:18:51.058+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:18:51.058+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:18:51.059+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:18:51.059+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:18:51.584+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:18:51.650+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:18:51.650+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:18:51.658+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:18:51.658+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:18:51.672+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.644 seconds
[2024-11-24T20:19:23.894+0000] {processor.py:161} INFO - Started process (PID=1078) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:19:23.895+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:19:23.897+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:19:23.896+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:19:23.917+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:19:23.917+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:19:23.918+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:19:23.918+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:19:23.919+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:19:23.919+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:19:24.392+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:19:24.456+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:19:24.455+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:19:24.463+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:19:24.463+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:19:24.478+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.589 seconds
[2024-11-24T20:19:56.742+0000] {processor.py:161} INFO - Started process (PID=1089) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:19:56.742+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:19:56.744+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:19:56.744+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:19:56.763+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:19:56.763+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:19:56.764+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:19:56.764+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:19:56.765+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:19:56.765+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:19:57.275+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:19:57.339+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:19:57.338+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:19:57.346+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:19:57.346+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:19:57.360+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.623 seconds
[2024-11-24T20:20:30.188+0000] {processor.py:161} INFO - Started process (PID=1100) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:20:30.188+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:20:30.190+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:20:30.190+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:20:30.210+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:20:30.210+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:20:30.211+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:20:30.211+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:20:30.212+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:20:30.212+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:20:30.815+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:20:30.879+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:20:30.879+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:20:30.886+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:20:30.886+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:20:30.904+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.721 seconds
[2024-11-24T20:21:02.032+0000] {processor.py:161} INFO - Started process (PID=1111) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:21:02.033+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:21:02.034+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:21:02.034+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:21:02.054+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:21:02.054+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:21:02.055+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:21:02.055+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:21:02.056+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:21:02.056+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:21:02.512+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:21:02.580+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:21:02.580+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:21:02.588+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:21:02.588+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:21:02.601+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.574 seconds
[2024-11-24T20:21:34.890+0000] {processor.py:161} INFO - Started process (PID=1122) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:21:34.890+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:21:34.892+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:21:34.892+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:21:34.912+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:21:34.912+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:21:34.913+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:21:34.913+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:21:34.914+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:21:34.914+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:21:35.326+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:21:35.393+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:21:35.393+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:21:35.400+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:21:35.400+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:21:35.417+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.532 seconds
[2024-11-24T20:22:07.747+0000] {processor.py:161} INFO - Started process (PID=1133) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:22:07.748+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:22:07.750+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:22:07.749+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:22:07.771+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:22:07.770+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:22:07.772+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:22:07.771+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:22:07.773+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:22:07.773+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:22:08.313+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:22:08.377+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:22:08.376+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:22:08.384+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:22:08.384+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:22:08.399+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.656 seconds
[2024-11-24T20:22:40.149+0000] {processor.py:161} INFO - Started process (PID=1144) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:22:40.150+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:22:40.151+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:22:40.151+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:22:40.174+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:22:40.174+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:22:40.176+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:22:40.175+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:22:40.177+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:22:40.177+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:22:40.702+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:22:40.765+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:22:40.765+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:22:40.772+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:22:40.772+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:22:40.787+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.642 seconds
[2024-11-24T20:23:13.101+0000] {processor.py:161} INFO - Started process (PID=1155) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:23:13.102+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:23:13.103+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:23:13.103+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:23:13.123+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:23:13.123+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:23:13.124+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:23:13.124+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:23:13.125+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:23:13.125+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:23:13.640+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:23:13.705+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:23:13.705+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:23:13.713+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:23:13.713+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:23:13.729+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.633 seconds
[2024-11-24T20:23:45.580+0000] {processor.py:161} INFO - Started process (PID=1166) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:23:45.580+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:23:45.582+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:23:45.582+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:23:45.605+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:23:45.604+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:23:45.606+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:23:45.606+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:23:45.607+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:23:45.607+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:23:46.044+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:23:46.114+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:23:46.114+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:23:46.122+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:23:46.121+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:23:46.138+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.564 seconds
[2024-11-24T20:24:18.368+0000] {processor.py:161} INFO - Started process (PID=1177) to work on /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:24:18.369+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/historical_data_etl.py for tasks to queue
[2024-11-24T20:24:18.371+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:24:18.370+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:24:18.390+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:24:18.390+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_nba' for task execution.
[2024-11-24T20:24:18.391+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:24:18.391+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-11-24T20:24:18.392+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:24:18.392+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-11-24T20:24:18.855+0000] {processor.py:840} INFO - DAG(s) 'nba_games_etl' retrieved from /opt/airflow/dags/historical_data_etl.py
[2024-11-24T20:24:18.924+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:24:18.923+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-11-24T20:24:18.932+0000] {logging_mixin.py:188} INFO - [2024-11-24T20:24:18.932+0000] {dag.py:3954} INFO - Setting next_dagrun for nba_games_etl to None, run_after=None
[2024-11-24T20:24:18.947+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/historical_data_etl.py took 0.583 seconds
